core/catalog-core-cachecommands/pom.xml

```diff
@@ -78,7 +78,6 @@
         <dependency>
             <groupId>org.geotools</groupId>
             <artifactId>gt-cql</artifactId>
```

Comment:
Assume we just want to up it here and leave the managed version alone?  New ticket to up the version globally?